### PR TITLE
Release 29

### PR DESCRIPTION
**Pull request by lucasmoura**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,28 @@
+ubuntu-advantage-tools (29) mantic; urgency=medium
+
+  * d/README:
```
Comment (renanrodrigo) [inline debian/changelog:3]: I think all `d/` changes should be listed - including `control`, `postinst`, and `postrm` - even if the changes are simple enough or not user-related, we have been instructed in the past to make those explicit.

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+    - api
+      + new endpoint: u.pro.security.fix.plan.cve.v1
+      + new endpoint: u.pro.security.fix.plan.usn.v1
+      + new endpoint: u.apt_news.current_news.v1
+      + add more data explaining the decision made for
+        u.pro.security.status.reboot_required.v1 endpoint
+    - contract:
+      + send activityInfo after cli attach, enable and disable
+      + start recording when the machine has attached to a Pro subsctiption
+    - esm: create static file to pin packages from esm-infra and esm-apps with
```
Comment (renanrodrigo) [inline debian/changelog:16]:
```suggestion
    - esm: create static files to pin packages from esm-infra and esm-apps with
```

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+           higher priority (GH: #2580)
+    - fix: ignore non-USN items on related usns
+    - landscape: add support for the service
+    - logging: update log names
```
Comment (renanrodrigo) [inline debian/changelog:20]:
```suggestion
    - logging: update logger names
```

**Comment by lucasmoura** [comment]
@renanrodrigo updated

**Review by renanrodrigo — APPROVED** [review]

File: debian/changelog [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+      + add more data explaining the decision made for
+        u.pro.security.status.reboot_required.v1 endpoint
+    - contract:
+      + send activityInfo after cli attach, enable and disable
+      + start recording when the machine has attached to a Pro subsctiption
+    - esm: create static files to pin packages from esm-infra and esm-apps with
+           higher priority (GH: #2580)
+    - fix: ignore non-USN items on related usns
+    - landscape: add support for the service
+    - logging: update logger names
```
Comment (lucasmoura) [inline debian/changelog:26]: logging: update to ubuntupro logging namespace

File: debian/changelog [hunk 5]
```diff
@@ -1,3 +1,34 @@
...
+      + new endpoint: u.pro.security.fix.plan.cve.v1
+      + new endpoint: u.pro.security.fix.plan.usn.v1
+      + new endpoint: u.apt_news.current_news.v1
+      + add more data explaining the decision made for
+        u.pro.security.status.reboot_required.v1 endpoint
+    - contract:
+      + send activityInfo after cli attach, enable and disable
+      + start recording when the machine has attached to a Pro subsctiption
+    - esm: create static files to pin packages from esm-infra and esm-apps with
+           higher priority (GH: #2580)
```
Comment (lucasmoura) [inline debian/changelog:23]: Double check identation

File: debian/changelog [hunk 6]
```diff
@@ -1,3 +1,34 @@
...
+    - anbox-cloud: add support for service
+    - api
+      + new endpoint: u.pro.security.fix.plan.cve.v1
+      + new endpoint: u.pro.security.fix.plan.usn.v1
+      + new endpoint: u.apt_news.current_news.v1
+      + add more data explaining the decision made for
+        u.pro.security.status.reboot_required.v1 endpoint
+    - contract:
+      + send activityInfo after cli attach, enable and disable
+      + start recording when the machine has attached to a Pro subsctiption
```
Comment (lucasmoura) [inline debian/changelog:21]: more reliable detection when running on a docker container

**Review by orndorffgrant — APPROVED** [review]

**Pull request merged**